rotkehlchen/accounting/structures/base.py

```diff
@@ -119,6 +127,22 @@ def __init__(
         self.notes = notes
         self.identifier = identifier
 
+        # Check that the received event type and subtype is a valid combination
+        if (
+            __debug__ and
+            (
```

Comment:
as for what lefteris explained me it has to be 

```suggestion
        if __debug__:
```

or otherwise it won't be removed on production code